src/App.tsx

```diff
@@ -1,29 +1,74 @@
-import { useSelector } from 'react-redux';
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
```

Comment:
you can just format imports instead
```suggestion
import React,
{
  useEffect,
  useState,
} from 'react';
```